erpnext/stock/doctype/delivery_trip/delivery_trip.py

```diff
@@ -330,6 +334,22 @@ def get_directions(route, optimize):
 
 	return directions[0] if directions else False
 
+@frappe.whitelist()
+def validate_unique_delivery_notes(delivery_stops):
+	delivery_stops = json.loads(delivery_stops)
+	delivery_notes = [stop.get("delivery_note") for stop in delivery_stops if stop.get("delivery_note")]
+
+	if not delivery_notes:
+		return []
+
+	existing_trips = frappe.get_all("Delivery Stop",
+									filters={"delivery_note": ["IN", delivery_notes],
```

Comment:
tab formatting seems off here to me, pushed way to the right.

Reply:
Oh, might be Github's rendering. I use the Refined Github extension, which replaces tabs with spaces, I think, so the whitespace is much saner.